ai_transform/engine/abstract_engine.py

```diff
@@ -65,7 +65,11 @@ def __init__(
                             warnings.warn(f"Not all fields were found. Missing {field}")
 
         self._dataset = dataset
+
+        if any(["_chunk_" in field for field in select_fields]):
+            select_fields += ["_chunk_"]
```

Comment:
Shouldn't this be the field that _chunk_ is in and not _chunk_ itself

Reply:
```
   selected_fields += [field]
```

Reply:
```
for field in select_fields:
   if "_chunk_" in field: 
       index = field.index("_chunk_") + len("_chunk_")
       chunk_field = field[:index] 
        select_fields += field[:_chunk_]
```

Reply:
If a user specifies `sentence_chunk_.value` 
it should add `sentence_chunk_` instead